diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/hooks/useOnScreen.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/hooks/useOnScreen.tsx
index 469e773..7707740 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/hooks/useOnScreen.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/hooks/useOnScreen.tsx
@@ -2,7 +2,7 @@ import { useState, useEffect, MutableRefObject } from 'react';
 
 // https://usehooks.com/useOnScreen/
 
-export function useOnScreen<T>(ref: MutableRefObject<T | undefined>, threshold = 0.25): boolean {
+export function useOnScreen<T>(ref: MutableRefObject<T | undefined>, threshold = 0.01): boolean {
     const [isIntersecting, setIntersecting] = useState(false);
 
     useEffect(() => {
